allofplos/transformations.py

```diff
@@ -23,28 +18,89 @@
 annotation = 'annotation'
 correction = 'correction'
 ANNOTATION_URL = 'http://journals.plos.org/plosone/article/file?id=10.1371/annotation/'
-annotation_url_int = 'http://contentrepo.plos.org:8002/v1/objects/mogilefs-prod-repo?key=10.1371/annotation/'
 ANNOTATION_DOI = '10.1371/annotation'
-BASE_URL_ARTICLE_LANDING_PAGE = 'http://journals.plos.org/plos{}/article?id='
+BASE_URL_ARTICLE_LANDING_PAGE = 'http://journals.plos.org/plos{}/article?id={}'
+BASE_URL_LANDING_PAGE = 'http://journals.plos.org/plos{}/'
+LANDING_PAGE_SUFFIX = '{}?id={}'
+
+plos_page_dict = {'article': 'article',
+                  'asset': 'article/asset',
+                  'articleFigsAndTables': 'article/assets/figsAndTables',
+                  'articleAuthors': 'article/authors',
+                  'citationDownloadPage': 'article/citation',
+                  'downloadBibtexCitation': 'article/citation/bibtex',
+                  'downloadRisCitation': 'article/citation/ris',
+                  'figuresPage': 'article/figures',
+                  'assetFile': 'article/file',
+                  'assetXMLFile': 'article/file',
+                  'articleMetrics': 'article/metrics',
+                  'articleRelated': 'article/related'}
+
+
+def _get_base_page(journal):
+    """Make the base of a PLOS URL journal-specific.
+    
+    Defaults to PLOS ONE.
+
+    Use in conjunction with `get_page()` in the Article class.
+    """
+    journal_map = OrderedDict([
```

Comment:
Technically this doesn't need to be an `OrderedDict` since we're not iterating through it, but are just grabbing the key. I'm fine with it remaining as is or changing to be a regular dict. As you wish.

Reply:
I second that, order is not required here.